src/devices/README.md

```diff
@@ -45,6 +45,7 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 * [Servomotor](Servo/README.md)
 * [ADXL345 - Accelerometer](Adxl345/README.md)
 * [DHTxx - Digital-Output Relative Humidity & Temperature Sensor Module](Dhtxx/README.md)
+* [M×N Matrix Keyboard](MatrixKeyboard/README.md)
```

Comment:
Is it possible to add here the actual chip number of the keyboard?

Reply:
Maybe not. This is a module, not a chip. It doesn't have a exactly number.

Reply:
I'd vote for the pic since it's not obvious what this device is immediately (i.e. I couldn't find it on Amazon)

Reply:
FYI: don't edit this file by hand at all, we have a script for that (tools/device-listing; just dotnet run after you cd into the directory - note I didn't test the tool on Windows but expecting it to work)